services/app-web/src/pages/ReplaceRate/ReplaceRate.tsx

```diff
@@ -0,0 +1,80 @@
+import { GridContainer } from '@trussworks/react-uswds'
+import React, { useEffect, useState } from 'react'
+import { useNavigate, useParams } from 'react-router-dom'
+import { Loading } from '../../components'
+import { usePage } from '../../contexts/PageContext'
+import { useFetchRateQuery } from '../../gen/gqlClient'
+import styles from '../SubmissionSummary/SubmissionSummary.module.scss'
+import { GenericErrorPage } from '../Errors/GenericErrorPage'
+import { useAuth } from '../../contexts/AuthContext'
+import { Error404 } from '../Errors/Error404Page'
+
+export const ReplaceRate = (): React.ReactElement => {
+    // Page level state
+    const { loggedInUser } = useAuth()
+    const { updateHeading } = usePage()
+    const navigate = useNavigate()
+    const [rateName, setRateName] = useState<string | undefined>(undefined)
+    const { rateID } = useParams()
+    if (!rateID) {
+        throw new Error('PROGRAMMING ERROR: rateID param not set')
+    }
+
+    useEffect(() => {
+        updateHeading({ customHeading: rateName })
+    }, [rateName, updateHeading])
+
+    const { data, loading, error } = useFetchRateQuery({
+        variables: {
+            input: {
+                rateID,
+            },
+        },
+    })
+
+    const rate = data?.fetchRate.rate
+    const currentRateRev = rate?.revisions[0]
+
+    if (loading) {
```

Comment:
Reminder: Theres a ErrorOrLoading compnent that handles a lot of this - also handles forbidden (which I think we should use here instead of a redirect)